mongodbatlas/data_source_mongodbatlas_data_lake_pipeline_test.go

```diff
@@ -1,12 +1,14 @@
 package mongodbatlas
```

Comment:
@Zuhairahmed @andreaangiolillo @AgustinBettati @lantoli @maastha 

We are deprecating data_lake[s] but the data_lake_pipeline[s] were not deprecated so they are actually staying. 

Is this correct?

Reply:
Looking at this issue https://jira.mongodb.org/browse/INTMDB-806 its seems correct.

Reply:
Correct, you can see Data Lake Pipelines is a seperate endpoint in latest Atlas Admin API and was not deprecated there either: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Data-Lake-Pipelines